### DISCUSSION TITLE
No new release since january, why ?

### DISCUSSION THREAD

**Discussion by tomS3210** [teslamate-org/teslamate#2747] [General]
Hello Teslamate teams,

I can see in the release note that there is a lot of improvment availables but since January they are "unreleased" 

Can you explain why ? this is not a criticism but to understand.

Regards
Thomas

**Comment by cwanja** [comment]
Fixed, new release today!

**Reply by tomS3210** [reply]
Thanks a lot :-)

**Reply by cwanja** [reply]
I did nothing, all @adriankumpf 😊 just reporting the news hah.

**Reply by tomS3210** [reply]
So a usual, thanks to @adriankumpf ;-)

**Reply by ktsenvolt** [reply]
Not sure why I did not get a notification of the new release on TeslaMate .
![image](https://user-images.githubusercontent.com/80360674/179218377-9b73c1be-0a16-4798-8e9b-0dc1fd2c087d.png)

**Comment by djpalmis** [comment]
I start to think that @adriankumpf lost interest in the project 😔
